### PR TITLE
spec: add Fedora-specific Requires

### PR DESCRIPTION
**Pull request by smithfarm**
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: sesdev.spec [hunk 1]
```diff
@@ -38,13 +38,18 @@ BuildRequires:  fdupes
...
 Requires:       python3-libvirt-python >= 5.1.0
-Requires:       python3-pycryptodomex >= 3.4.6
 Requires:       python3-PyYAML >= 3.13
-Requires:       python3-setuptools
 %endif
+%if 0%{?fedora}
+Requires:       python3-jinja2 >= 2.10.1
+Requires:       python3-libvirt >= 5.1.0
+Requires:       python3-pyyaml >= 3.13
+%endif
```
Comment (smithfarm) [inline sesdev.spec]: Please note: this is on purpose not implemented as an `%else` block. Having a separate conditional block for Fedora is IMO easier to understand and makes it explicit that we are supporting Fedora.

**Comment by Conan-Kudo** [comment]
@smithfarm This change was unnecessary, as [the dependencies were being auto-generated by the Python module dependency generator](https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_automatically_generated_dependencies), which has been [enabled by default since Fedora 30](https://fedoraproject.org/wiki/Changes/EnablingPythonGeneratorsByDefault).

**Comment by Conan-Kudo** [comment]
@smithfarm As a consequence of this change, you now have duplicate dependencies described: https://build.opensuse.org/package/binary/filesystems:ceph/sesdev/Fedora_30/x86_64/sesdev-1.1.5+1582717868.g68df753-11.1.noarch.rpm

**Comment by smithfarm** [comment]
@Conan-Kudo Thanks! After giving the matter some further thought, I prefer to disable the automatic dependency generation. See https://github.com/SUSE/sesdev/pull/140 for the rationale.

**Pull request merged**